CASSANDRA-17486 CQLSH broken as per the quick start documentation

sathyakplm opened the pull request.

dominem commented:
This PR really helped me with the quickstart. Thank you @sathyakplm!

I was stuck on step 4:
```fish
dm@dm-pc ~> docker run --rm --network cassandra -v (pwd)"/data.cql:/scripts/data.cql" -e CQLSH_HOST=cassandra -e CQLSH_PORT=9042 nuvo/docker-cqlsh
Checking connection to cassandra...
Can't establish connection, will retry again in 1 sconds
Can't establish connection, will retry again in 2 sconds
^CCan't establish connection, will retry again in 3 sconds
Can't establish connection, will retry again in 4 sconds
Can't establish connection, will retry again in 5 sconds
Failed to connect to cassandra at cassandra:9042
```

With `-e CQLVERSION=3.4.5` all went well:
```fish
dm@dm-pc ~> docker run --rm --network cassandra -v (pwd)"/data.cql:/scripts/data.cql" -e CQLSH_HOST=cassandra -e CQLSH_PORT=9042 -e CQLVERSION=3.4.5 nuvo/docker-cqlsh
Checking connection to cassandra...
Connected to Test Cluster at cassandra:9042.
Executing /scripts/data.cql...
Done.
```

ErickRamirezAU committed:
DOC - Fixed CQL version for Docker cqlsh command in Quickstart guide

patch by Sathyanarayanan Saravanamuthu, Erick Ramirez; reviewed by Erick Ramirez for CASSANDRA-17486

Co-authored by: Sathyanarayanan Saravanamuthu <sathya.kplm@outlook.com>
Co-authored by: Erick Ramirez <erickramirezau@gmail.com>

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/quickstart.adoc:
```diff
@@ -107,15 +107,15 @@ Much like an SQL shell, you can also of course use CQLSH to run CQL commands int
 --
 Connected to Test Cluster at cassandra:9042.
-[cqlsh 5.0.1 | Cassandra 3.11.10 | CQL spec 3.4.4 | Native protocol v4]
+[cqlsh 5.0.1 | Cassandra 4.0.4 | CQL spec 3.4.5 | Native protocol v5]
```
I've fixed the versions in the sample cqlsh output.
